### PR TITLE
add BRB-seqTools

### PR DESCRIPTION
**Pull request by tomkinsc**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Details:
https://www.biorxiv.org/content/biorxiv/early/2018/01/30/256594.full.pdf
https://github.com/DeplanckeLab/BRB-seqTools

**HOLD** merge until license is clarified: https://github.com/DeplanckeLab/BRB-seqTools/issues/2

### CODE REVIEW

File: recipes/brb-seqtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{% set version = "1.1" %}
+{% set name = "BRB-seqTools" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+build:
+  number: 0
```
Comment (epruesse) [inline recipes/brb-seqtools/meta.yaml]: add `noarch: python`

**Comment by druvus** [comment]
@epruesse the python wrapper around jar files is described in the [Bioconda guidelines](https://bioconda.github.io/guidelines.html#java)

**Comment by epruesse** [comment]
@druvus huh - never saw that

Wonder whether the wrapper should be its own package “jar-runner” or so, with a “make-jar-entrypoint” type script.

**Comment by tomkinsc** [comment]
Yeah, the license issue is known and noted in the PR comment. I have an issue outstanding upstream requesting that the package author choose (and include) a license. We're waiting.

File: recipes/brb-seqtools/meta.yaml (hunk 1 above)
Comment (epruesse) [reply recipes/brb-seqtools/meta.yaml]: Yes, ignore the comment. We don't do `noarch` yet. I was confused with conda-forge requirements. There, "universal" python packages, i.e. pure python compatible with all versions, are built with `noarch` as they don't need selectors and work with current `conda-build` already.

File: recipes/brb-seqtools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    - python
+    - openjdk >=8
+
+test:
+  commands:
+    - BRBseqTools
+
+about:
+  home: https://github.com/DeplanckeLab/BRB-seqTools
+  license: TBD
```
Comment (epruesse) [inline recipes/brb-seqtools/meta.yaml]: Hold until  https://github.com/DeplanckeLab/BRB-seqTools/issues/2 is fixed

Comment (tomkinsc) [reply recipes/brb-seqtools/meta.yaml]: The author of `BRB-seqTools` selected a Creative Commons [BY-NC-SA](https://github.com/DeplanckeLab/BRB-seqTools/blob/master/LICENSE.md) license. 
@bioconda/core: Does the non-commercial constraint prohibit inclusion in Bionconda? Creative Commons provides [this guidance](https://wiki.creativecommons.org/wiki/NonCommercial_interpretation) on the NonCommercial element.

**Comment by tomkinsc** [comment]
The author of `BRB-seqTools` selected a Creative Commons [BY-NC-SA](https://github.com/DeplanckeLab/BRB-seqTools/blob/master/LICENSE.md) license. 
@bioconda/core: What do we say; does the non-commercial constraint prohibit inclusion in Bionconda? Creative Commons provides [this guidance](https://wiki.creativecommons.org/wiki/NonCommercial_interpretation) on the NonCommercial element.

**Comment by epruesse** [comment]
Can you ask the authors whether they consider distribution via Bioconda in violation to the license? Ideally, they'd amend the license with a clarification, explicitly giving us permission to build packages and distribute via Anaconda Cloud. That way, we don't have to ponder.

**Comment by daler** [comment]
Closing due to inactivity, please re-open if needed.

**Pull request closed without merging**